torch/csrc/profiler/collection.cpp

```diff
@@ -850,14 +850,20 @@ void calculate_unique_tensor_ids(std::vector<result_ptr_t>& sorted_results) {
     };
 
     ska::flat_hash_set<storage_id_t> tensor_set;
+    auto insert_tensor = [&lookup, &tensors, &tensor_set](TensorMetadata& m) {
+      if (m.impl_ && m.data_) {
+        const auto id = lookup(m.data_);
```

Comment:
That's a fair point. You're right, but I'm going to leave it simply because https://github.com/pytorch/pytorch/pull/87133 (later in the stack) removes it altogether.